src/ui/public/agg_types/metrics/latest.js

```diff
@@ -0,0 +1,44 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
```

Comment:
Don't feel compelled to be so verbose.  When you see extremely long variable names like this in other imports, it was probably the result of an automatic transformation that we applied to the whole codebase.  We tend to shorten them when we're modifying those files, in fact.  Something like `MetricAggTypeProvider` would probably be better.